platform-sdk/docs/proposals/TSS-Library/native/Architecture.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.pairings.nativ3support;
+
+public enum Architecture {
+    AMD64("amd64"),
+    ARM("arm"),
+    ARM64("arm64"),
+    I386("i386"),
+    PPC64LE("ppc64le"),
+    S390X("s390x");
```

Comment:
Probably shouldn't define PPC, S390, or I386.

Reply:
This class was taken from existent code, but yeah, lets keep only the values we will support